package.json

```diff
@@ -1,37 +1,39 @@
 {
-  "name": "videojs5-hlsjs-source-handler",
+  "name": "videojs-hlsjs-source-handler",
+  "version": "0.2.2-beta.0",
```

Comment:
This kind of warrants a version reset (different name) but not really matter to me.

Reply:
Well let's bump it to 0.3, since we have quite a few changes here